api/api.go

```diff
@@ -26,14 +26,21 @@ var (
 	// metric api.iters_to_points is how long it takes to decode points from a chunk iterator
 	itersToPointsDuration = stats.NewLatencyHistogram15s32("api.iters_to_points")
 
-	// metric api.request_handle is how long it takes to handle a render request
-	reqHandleDuration = stats.NewLatencyHistogram15s32("api.request_handle")
-
 	// metric api.requests_span.mem_and_cassandra is the timerange of requests hitting both in-memory and cassandra
 	reqSpanBoth = stats.NewMeter32("api.requests_span.mem_and_cassandra", false)
 
 	// metric api.requests_span.mem is the timerange of requests hitting only the ringbuffer
 	reqSpanMem = stats.NewMeter32("api.requests_span.mem", false)
+
+	// metric api.render.series is the number of series a /render request is handling.  This is the number
+	// of metrics after all of the targets in the request have expaned by searching the index.
+	reqRenderSeriesCount = stats.NewMeter32("api.render.series", false)
+
+	// metric api.render.series is the number of targets a /render request is handling.
+	reqRenderTargetCount = stats.NewMeter32("api.render.targets", false)
+
+	// metric api.render.series is the number of points that need to be fetched for a /render request.
+	reqRenderPointsCount = stats.NewMeter32("api.render.points", false)
```

Comment:
note that alignRequests also emits the following metrics:
```
	numPointsFetch  = stats.NewMeter32("api.align_requests.num_points_fetching", false)
	numPointsReturn = stats.NewMeter32("api.align_requests.num_points_returning", false)
	chosenArchive   = stats.NewMeter32("api.align_requests.chosen_archive", false)
```

numPointsFetch is exactly the same as reqRenderPointsCount I think. it would be nice to unify these metrics somehow.